test/integration/local/test_python_service.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import os
+import shutil
+import subprocess
+import sys
+import time
+
+import pytest
+
+import requests
+
+PING_URL = 'http://localhost:8080/ping'
+INVOCATIONS_URL = 'http://localhost:8080/invocations'
+
+
+@pytest.fixture(scope='session', autouse=True, params=['input_output_handler', 'handler'])
+def volume(tmpdir_factory, request):
+    try:
+        script_file = 'test/integration/local/script/{}_example.py'.format(request.param)
+        requirements_file = 'test/integration/local/script/requirements_example.txt'
+        shutil.copy(script_file, tmpdir_factory.getbasetemp().join('inference.py'))
+        shutil.copy(requirements_file, tmpdir_factory.getbasetemp().join('requirements.txt'))
+
+        model_dir = os.path.abspath('test/resources/models')
```

Comment:
model_dir should be a temp dir outside the project root, since we are writing files to it now. to make that work, we'll need to copy the models from test/resources/models into the tempdir